core/src/main/java/com/netease/arctic/data/file/FileNameGenerator.java

```diff
@@ -223,6 +223,23 @@ public static long parseTransactionId(String path) {
     }
   }
 
+  /**
+   * Parse transaction id of change file.
+   *
+   * @param path path
+   * @return transactionId, return 0 if path is not arctic file format.
+   */
+  public static long parseChangeTransactionId(String path, long sequenceNumber) {
```

Comment:
Can we implement this method dependent on `parseTransactionId`?

Reply:
Ok, I will change it.